### ISSUE TITLE
[nushell] Enter doesn't execute immediately

### ISSUE THREAD

**Issue by gergelyk** [atuinsh/atuin#1392]
**Describe the bug**
Hitting Enter and Alt+Enter gives the same effect  - selects item and edits.

**To Reproduce**
1. CTRL+R
2. Select item
3. Hit Enter

**Expected behavior**
Command is executed immediately.

**Device (please complete the following information):**
 - OS: linux
 - Shell: nu 0.86.0
 - atuin Version: 17.0.1

**Additional context**
atuin enabled by:
```
atuin init nu | save ~/.config/nushell/atuin.nu
source ~/.config/nushell/atuin.nu
```

**Comment by arcuru** [comment]
`enter_accept` is not supported on nushell: https://atuin.sh/docs/config/#enter_accept

To support it, Nushell needs to have a feature equivalent to ZSH's `zle accept-line`. If they have that then we could add support for it here fairly easily.

**Comment by gergelyk** [comment]
Thanks @arcuru. Feature request added.

**Comment by mrdgo** [comment]
I created a [pr](https://github.com/nushell/nushell/pull/13728) to add this feature, checkout my fork of both nushell and reedline to play around with it.

**Comment by dougg0k** [comment]
@mrdgo They requested a cleanup in the PR to get merged. Maybe you didnt see that.

**Comment by mrdgo** [comment]
Yes, I saw that. I'll do it as soon as I find the time for a few hours of coding 😅

**Comment by stuartcarnie** [comment]
Hey all, I've created a [PR for nushell](https://github.com/nushell/nushell/pull/16193) for this, and [implemented an updated `atuin` completer script](https://github.com/nushell/nushell/pull/16193#issuecomment-3082100115) that works just like ZSH.

**Comment by stuartcarnie** [comment]
Update: The [PR has merged](https://github.com/nushell/nushell/pull/16193), so the next version of Nushell will support immediately accepting the command-line. I have been using it with Atuin for about a month now, and it works great.

**Comment by exsesx** [comment]
Awesome, been waiting for this to land. I pulled the latest master of Nushell right away and set it up, and it’s working great with Atuin on my end too, amazing.

**Comment by mrdgo** [comment]
After almost a year! Thanks for cleaning up the PR 👍

**Comment by wjarka** [comment]
Does it require any additional configuration in Atuin? I am on 0.107 which supposedly adds that PR and it still doesn't work for me

**Comment by LovesTha** [comment]
I'm also having trouble getting it working. 
Atuin: 18.10.0
Nu: 0.108.0
Both are new enough?

I have added `enter_accept = true` to my config file, and I still can't get it to work.

**Comment by shgew** [comment]
~~Same here 👋🏻 
@stuartcarnie @exsesx could you please explain how to get it working? :)~~

Nevermind, got it working. Turns out [enter_accept](https://docs.atuin.sh/configuration/config/#enter_accept) is `false` by default. Whoops. Sorry for pinging you for no reason.

**Comment by shunichironomura** [comment]
> I'm also having trouble getting it working. Atuin: 18.10.0 Nu: 0.108.0 Both are new enough?
> 
> I have added `enter_accept = true` to my config file, and I still can't get it to work.

I had the same issue, but in my case I just forgot to run
```nu
atuin init nu | save ~/.local/share/atuin/init.nu -f
```
to update Atuin's `init.nu`.

**Comment by LovesTha** [comment]
Unsure what fixed mine, but it could have been a roundabout way of doing that.
